src/adapters/lokijs/worker/encodeQuery/index.js

```diff
@@ -86,6 +87,18 @@ const noNullComparisons: OperatorFunction => OperatorFunction = operator => valu
   $and: [operator(value), weakNotEqual(null)],
 })
 
+const like: OperatorFunction = value => {
+  if (typeof value === 'string') {
+    return {
+      $regex: new RegExp(value.replace(/%/g, '.*').replace('_', '.'), 'i'),
+    }
+  }
+
+  return {
+    $regex: new RegExp('!.*', 'i'),
```

Comment:
Hmm I am not sure what is the purpose of this? If `value ` is not a string we want to match every word starting from `!`? Maybe I don't understand something :)

Reply:
Good catch! I want to match nothing, but it's not the good way 😅.

My suggestion:
```suggestion
    $regex: new RegExp('(?!.*)', 'i'),
```